File: Back/securecapita/securecapita/src/main/java/io/getarrayus/securecapita/DTO/PaypalDTO.java
package io.getarrayus.securecapita.DTO;

import lombok.Data;
import lombok.Value;

@Data
public class PaypalDTO {


        Double total;
        String currency;
        String method;
        String intent;


    public Double getTotal() {
        return total;
    }

    public String getCurrency() {
        return currency;
    }

    public String getIntent() {
        return intent;
    }

    public String getMethod() {
        return method;
    }
}

File: front/src/app/klasy/paypal-dto.ts
export class PaypalDTO {
    total: number;
    currency: string;
    method: string;
    intent: string;
  
    constructor(total:number) {
      this.total = total;    
      this.currency = 'USD'; 
      this.method = 'paypal';
      this.intent = 'sale';   
    }
  }
  